### PR TITLE
Fix group for VEI issuers

### PR DESCRIPTION
**Pull request by irbekrm**
This fixes a typo where the wrong API group was being used in a command that lists Venafi Enhanced issuers in cluster.

Signed-off-by: irbekrm <irbekrm@gmail.com>

### CODE REVIEW

File: internal/kubernetes/status/status.go [hunk 1]
```diff
@@ -394,7 +394,7 @@ func findIssuers(ctx context.Context, cfg *rest.Config) ([]summaryIssuer, error)
 			}
 			for _, issuer := range issuers.Items {
 				summaryIssuers = append(summaryIssuers, summaryIssuer{
-					APIVersion: kmsissuerv1alpha1.GroupVersion.String(),
+					APIVersion: veiv1alpha1.SchemeGroupVersion.Group,
```
Comment (inteon) [inline internal/kubernetes/status/status.go:397]: @irbekrm why is this just the group?

Comment (irbekrm) [reply internal/kubernetes/status/status.go]: Thanks for spotting this @inteon ! I can't remember any reason for using just the group, so most likely it was a typo.

Comment (irbekrm) [reply internal/kubernetes/status/status.go]: Fixed in https://github.com/jetstack/jsctl/pull/72/commits/25fdd77b7bcfce03f3b05c1b4c5fe9828d5df1d9

**Comment by inteon** [comment]
Thank you for the fix. The `policyCRDsFound` prevents an error in case `IncludeCertificateRequestPolicies` is true and the policy CRDs are not installed. @irbekrm can you add "writing a test to test for this error" to your backlog?
/lgtm
/approve

**Review by inteon — APPROVED** [review]
Thank you for the fix. The `policyCRDsFound` change prevents an error in case `IncludeCertificateRequestPolicies` is true and the policy CRDs are not installed. @irbekrm can you add "writing a test to test for this error" to your backlog?
/lgtm
/approve

**Pull request merged**